mapper/mapper.go

```diff
@@ -135,9 +129,8 @@ func CreateFilterableLandingPage(ctx context.Context, d dataset.Model, ver datas
 
 			var pDim datasetLandingPageFilterable.Dimension
 
-			var ok bool
 			var title string
-			if title, ok = dimensionTitleMapper[opt.Items[0].DimensionID]; !ok {
+			if len(opt.Items) > 0 {
```

Comment:
Is there a unit test for this that also needs updating?

Reply:
unit test for dimensionTitleMapper